bootstrap/kubeadm/controllers/kubeadmconfig_controller.go

```diff
@@ -735,7 +735,6 @@ func (r *KubeadmConfigReconciler) reconcileDiscovery(ctx context.Context, cluste
 		}
 
 		config.Spec.JoinConfiguration.Discovery.BootstrapToken.Token = token
-		log.Info("Altering JoinConfiguration.Discovery.BootstrapToken", "Token", token)
```

Comment:
Instead of removing the whole line, should we just remove the token instead?

Reply:
My bad, let me fix that